deploy/infra/aws/terraform/cloud-native/cluster-infra/input.tf

```diff
@@ -233,3 +233,51 @@ variable "rds_db_port" {
   default     = 5432
   description = "The port on which the RDS instance accepts connections"
 }
+
+###################################################################
+#               Cloud-pipeline script deployment
+###################################################################
+
+variable "deployment_id" {
+  type        = string
+  description = "Deployment Name"
+}
+
+variable "elb_public_subnet" {
+  type        = string
+  description = "Public subnet for Load Balancer"
+}
+
+variable "eipalloc" {
+  type        = string
+  description = "Ellastic IP id"
+}
+
+variable "ui_deployment_name" {
+  type        = string
+  description = "Deployment Name that will be shown as the browser tab name"
```

Comment:
`Deployment Name that will be shown as the browser tab name` -> `Deployment Name for the Cloud-Pipeline that will be used in several UI elements and appearance, f.i. it will be shown as the browser tab name`